droid-configs.inc

```diff
@@ -701,8 +705,12 @@ ln -s bluez5/bluetooth /etc/bluetooth   || :
 
 %package -n patterns-sailfish-device-configuration-common-%{rpm_device}
 Summary: Jolla Configuration common
```

Comment:
PersonallyI don't like these remove/disable variables, maybe have_sailfish_applications and make condition other way around?

Reply:
> 

Then we need to add have_sailfish_applications define in each droid-configs when git submodule is updated.

Reply:
Doesn't that defeat the purpose? We would make it this way the pattern of the port could just list: Requires: patterns-sailfish-applications.

Reply:
We already have `remove_modem` and `remove_bluetooth` because there are not many devices that don't have a modem/BT.
If we force each and every port (including community ports) to list its own capabilities, it will defeat the purpose of patterns in the first place.